base_local_planner/include/base_local_planner/geometry_math_helpers.h

```diff
@@ -64,6 +64,17 @@ Eigen::Vector3f twistToVector3f(const geometry_msgs::Twist& t);
 geometry_msgs::Pose vector3fToPose(const Eigen::Vector3f& vec);
 geometry_msgs::Twist vector3fToTwist(const Eigen::Vector3f& vec);
 
+double linearInterpolation(double value, double min_value, double max_value, double min_output, double max_output);
```

Comment:
Might want to make these inline, not that it matters but the change was more for readability and not repeating code.